omod/src/main/webapp/resources/htmlforms/089a-Daily-Activity-Laboratory-Register-for-Clinical-Chemistry.xml

```diff
@@ -1,4 +1,4 @@
-<htmlform formName="HMIS Form 089a  Daily Activity Laboratory Register for Clinical Chemistry"
```

Comment:
With this commit, you do not need this any more https://github.com/openmrs/openmrs-core/commit/add3b261d0db41d1a6e417058e9b63079bf97039